src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,101 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.ui.Ui;
+import duke.tasklist.TaskList;
+import duke.task.Todo;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.DukeException;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+
+/**
+ * Represents a Command object that will add new tasks.
+ */
+public class AddCommand extends Command {
+
+    private String taskType;
+    private String textInput;
+
+    /**
+     * Constructor for the AddCommand object.
+     *
+     * @param type The type of tasks to be added,
+     * (Todo, Deadline, Event).
+     * @param input The user's input containing the information
+     * of the task to be added.
+     */
+    public AddCommand(String type, String input) {
+        this.textInput = input;
+        this.taskType = type;
+    }
+
+    /**
+     * Signifies to the app to not terminate its current run.
+     */
+    @Override
+    public boolean isExit() {
+        return false;
+    }
+
+    /**
+     * Will add the provided task to the tasklist.
+     *
+     * @param stg The storage object to use file writing methods.
+     * @param ui The ui object to handle I/O requests.
+     * @param tasks The task list which holds all tasks available.
+     * @throws IOException If an I/O error occurs.
+     */
+    @Override
+    public String execute(Storage stg, Ui ui, TaskList tasks) throws IOException {
```

Comment:
According to [our coding standard](https://se-education.org/guides/conventions/java/intermediate.html), 

> In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)